matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/di/NetworkModule.kt

```diff
@@ -54,6 +55,13 @@ internal object NetworkModule {
         return CurlLoggingInterceptor()
     }
 
+    @MatrixScope
+    @Provides
+    @JvmStatic
+    fun providesApiInterceptor(): ApiInterceptor {
```

Comment:
This is not necessary to add provides method when you already have Inject annotation on the constructor of the classe. So you can remove those lines ^

Reply:
removed